DOCUMENTATION.md

```diff
@@ -156,19 +214,19 @@ We got you covered. You can declare a default header and use it when appropriate
 </config>
 ```
 
-### Splitting files
+## `<splitter>`
 
 Now that we now how to read the files, it's time to cut them ! Splitters are responsible for creating chunks from your
 source file. For exemple, if you have 100 sentences and you use the [Punctuation Splitter](#punctuation-splitter), these
```

Comment:
punctuation splitter is now regex splitter. My bad for this one.

Reply:
done. I also corrected the #